config/owasp/suppressions.xml

```diff
@@ -32,65 +30,32 @@
       CVEs are on transitive dependencies of springfox-swagger2 which hasn't been updated in about 2 years
       CVE-2022-22970 refer https://tools.hmcts.net/jira/browse/HMAN-266
       CVE-2022-22950 refer https://tools.hmcts.net/jira/browse/HMAN-316
+      CVE-2022-22968 refer https://tools.hmcts.net/jira/browse/CCD-3121
+      CVE-2022-22965 refer https://tools.hmcts.net/jira/browse/CCD-2926
+      CVE-2018-11039 refer https://tools.hmcts.net/jira/browse/CCD-3405
+      CVE-2018-11040 refer https://tools.hmcts.net/jira/browse/CCD-3407
+      CVE-2018-1257 refer https://tools.hmcts.net/jira/browse/CCD-3408
+      CVE-2020-5421 refer https://tools.hmcts.net/jira/browse/CCD-3409
+      CVE-2013-4152 refer https://tools.hmcts.net/jira/browse/HMAN-243
```

Comment:
Some of the old invalid CVEs (CVE-2014-0054, CVE-2013-7315 and CVE-2013-4152 etc) are excluded from the RSE dashboard stats. Do they still block the build if we don't have CVE suppressions?

Reply:
Local dependencyCheckAggregate fails if CVE-2013-4152, CVE-2013-7315, CVE-2014-0054 removed :-
```
spring-plugin-core-2.0.0.RELEASE.jar (pkg:maven/org.springframework.plugin/spring-plugin-core@2.0.0.RELEASE, cpe:2.3:a:vmware:spring_framework:2.0.0:release:*:*:*:*:*:*) : CVE-2013-4152, CVE-2013-7315, CVE-2014-0054
spring-plugin-metadata-2.0.0.RELEASE.jar (pkg:maven/org.springframework.plugin/spring-plugin-metadata@2.0.0.RELEASE, cpe:2.3:a:vmware:spring_framework:2.0.0:release:*:*:*:*:*:*) : CVE-2013-4152, CVE-2013-7315, CVE-2014-0054
```